.github/workflows/regression.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         config_set: [BaseMPI, ReverseMPI, ForwardMPI, BaseNoMPI, ReverseNoMPI, ForwardNoMPI, BaseOMP, ReverseOMP, ForwardOMP]
         include:
           - config_set: BaseMPI
-            flags: '-Denable-pywrapper=true -Denable-coolprop=true -Denable-tests=true --warnlevel=2 --werror'
+            flags: '-Denable-pywrapper=true -Denable-coolprop=true -Denable-mpp=true -Denable-tests=true --warnlevel=2 --werror'
```

Comment:
See what we do in the main meson.build for 'enable-coolprop' the error seems similar

Reply:
Yeah, Ill need to look into depth about this.  Compilation stuff is over my head...